crates/iota-graphql-rpc/tests/e2e_tests.rs

```diff
@@ -1004,12 +1033,192 @@ mod tests {
             .with_env()
             .init();
         let connection_config = ConnectionConfig::ci_integration_test_cfg();
-        let cluster =
-            iota_graphql_rpc::test_infra::cluster::start_cluster(connection_config, None).await;
+        let cluster = iota_graphql_rpc::test_infra::cluster::start_cluster(
+            connection_config,
+            None,
+            ServiceConfig::test_defaults(),
+        )
+        .await;
 
         cluster
             .wait_for_checkpoint_catchup(0, Duration::from_secs(10))
             .await;
         test_health_check_impl().await;
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn test_payload_total_exceeded() {
```

Comment:
Could we also test the edge case where user submits the maximum allowed transaction size?